### PR TITLE
Bazel updates for garden

### PR DESCRIPTION
**Pull request by mjcarroll**

### CODE REVIEW

**Review by iche033 — APPROVED** [review]
looks good to me. Built fine and tests passed with the `garden` branch in gz-bazel. Not sure why the abi checker job is failing.

**Comment by mjcarroll** [comment]
> Not sure why the abi checker job is failing

It's because I had this targeted to the test_cleanup branch initially, it's a checker problem.

**Comment by mjcarroll** [comment]
@osrf-jenkins retest this please

File: include/gz/physics/config.hh.in [hunk 1]
```diff
@@ -27,6 +27,7 @@
 
 #define GZ_PHYSICS_VERSION "${PROJECT_VERSION}"
 #define GZ_PHYSICS_VERSION_FULL "${PROJECT_VERSION_FULL}"
+#define GZ_PHYSICS_VERSION_NAMESPACE v${PROJECT_VERSION_MAJOR}
```
Comment (scpeters) [inline include/gz/physics/config.hh.in:30]: per https://github.com/gazebosim/gz-cmake/issues/412, this breaks ABI

it looks like this PR was merged even with a broken ABI build

**Pull request merged**